src/graphql/mirror.js

```diff
@@ -200,16 +207,36 @@ export class Mirror {
         .prepare("SELECT config FROM meta")
         .pluck()
         .get();
-      if (existingBlob === blob) {
+      if (existingBlob === configBlob) {
         // Already set up; nothing to do.
         return;
       } else if (existingBlob !== undefined) {
-        throw new Error(
-          "Database already populated with " +
-            "incompatible schema, options, or version"
-        );
+        const existingConfig: MirrorConfig = JSON.parse(existingBlob);
+        if (existingConfig.version !== mirrorConfig.version) {
+          throw new Error(
+            "Database already populated with incompatible version."
+          );
+        } else if (!deepEqual(existingConfig.schema, mirrorConfig.schema)) {
+          throw new Error(
+            "Database already populated with incompatible schema."
+          );
+        } else if (
+          !deepEqual(existingConfig.blacklistedIds, mirrorConfig.blacklistedIds)
+        ) {
+          console.error(dedent`\
+            Warning: Database already loaded with different blacklistedIds.\
+            Unexpected behavior may result. Consider resetting your cache.`);
+          // We don't need to do the rest of the initialization, because it was
+          // setup, albiet with different options.
```

Comment:
sp.: *albeit* (“although be it”)

Also, *set up* as a phrasal verb is two words.